.github/workflows/version-update.yml

```diff
@@ -0,0 +1,40 @@
+name: Update package version for release & hotfix branches
+
+on:
+  create:
+    branches: [release/*, hotfix/*]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+      - name: update package version
+        id: vars
+        run: |
+          BRANCH_NAME="${GITHUB_REF#refs/heads/}"
+          PACKAGE_VERSION="${GITHUB_REF##*/}"
+          echo ::set-output name=branch::${BRANCH_NAME}
+          echo ::set-output name=version::${PACKAGE_VERSION}
+          if [[ $PACKAGE_VERSION =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]
```

Comment:
Our release branches typically don't have a patch number, and I think we need to account for that (e.g. we'd want release/v1.11 to create a PR which updates the version to 1.11.0)

Reply:
I see, I will make the last patch number optional in the regex then